context/app/static/js/components/detailPage/related-entities/RelatedEntitiesSectionHeader/RelatedEntitiesSectionHeader.jsx

```diff
@@ -2,16 +2,23 @@ import React from 'react';
 import Button from '@material-ui/core/Button';
 
 import SectionHeader from 'js/shared-styles/sections/SectionHeader';
+import { useFlaskDataContext } from 'js/components/Contexts';
 import { SpacedSectionButtonRow } from 'js/shared-styles/sections/SectionButtonRow';
 
-function RelatedEntitiesSectionHeader({ header, searchPageHref, iconTooltipText }) {
+function RelatedEntitiesSectionHeader({ header, searchPageHref }) {
+  const {
+    entity: { entity_type },
+  } = useFlaskDataContext();
+
+  const tooltipTexts = {
```

Comment:
Same as for `provenanceTooltipText` this can be pulled outside of `RelatedEntitiesSectionHeader`.

Reply:
![image](https://github.com/hubmapconsortium/portal-ui/assets/45410262/0d698045-7de0-48b0-9163-8ff80ac8c0ee)